plugins/opusCodec/src/OpusEncoder.h

```diff
@@ -0,0 +1,68 @@
+#ifndef OPUSENCODER_H
+#define OPUSENCODER_H
+#include <plugins/CodecPlugin.h>
+#include "opus/opus.h"
+
+
+class AthenaOpusEncoder : public Encoder {
+public:
+    const int DEFAULT_BITRATE     = 128000;
+    const int DEFAULT_COMPLEXITY  = 10;
+    const int DEFAULT_APPLICATION = OPUS_APPLICATION_VOIP;
+    const int DEFAULT_SIGNAL      = OPUS_AUTO;
+
+
+    AthenaOpusEncoder(int sampleRate, int numChannels);
+    ~AthenaOpusEncoder() override;
+
+    virtual void encode(const QByteArray& decodedBuffer, QByteArray& encodedBuffer) override;
+
+
+    int getComplexity() const;
+    void setComplexity(int complexity);
+
+    int getBitrate() const;
+    void setBitrate(int bitrate);
+
+    int getVBR() const;
+    void setVBR(int vbr);
+
+    int getVBRConstraint() const;
+    void setVBRConstraint(int vbr_const);
+
+    int getMaxBandwidth() const;
+    void setMaxBandwidth(int maxbw);
+
+    int getBandwidth() const;
+    void setBandwidth(int bw);
+
+    int getSignal() const;
+    void setSignal(int signal);
+
+    int getApplication() const;
+    void setApplication(int app);
+
+    int getLookahead() const;
+
+    int getInbandFEC() const;
+    void setInbandFEC(int fec);
+
+    int getExpectedPacketLossPercent() const;
+    void setExpectedPacketLossPercent(int perc);
```

Comment:
Abbreviations in names should be avoided  CS1.2.12 https://github.com/kasenvr/project-athena/blob/kasen/core/CODING_STANDARD.md#1212-abbreviations-in-names-should-be-avoided


could be `expectedPacketLossPercentage`